.gitignore

```diff
@@ -1,4 +1,3 @@
 **/bin
-**/obj
```

Comment:
Thanks for adding prettier to this project. I just wonder if there is a reason to remove obj directory ?